versions/1.0.0.md

```diff
@@ -71,6 +79,16 @@ A URL representing the issueformat (allowing a different URL format to be swappe
 
 See [Substitutions]() for more details on substitutions.
 
+### userUrlFormat (`string`)
+
+A URL representing the a user's profile URL on GitHub, Gitlab, etc.
+
+```
+{{host}}/{{user}}
```

Comment:
This is for configuring the link(s) to `@username` references in commit messages, correct? Still digging through the code to find out where that happens currently... is it worth outlining that in the description somehow? At first, I thought it might have been related to the `{{host}}/{{owner}}` prefix on the other variables.